### PR TITLE
Update observability 

### PR DESCRIPTION
**Pull request by CathPag**
Signed-off-by: Catherine Paganini <74001907+CathPag@users.noreply.github.com>

### Describe your changes
This definition needs to reflect that observability is a system property. Observability tools are the tools that enable us to get those insights. 

I removed the three sections (not used for properties) and merged the content. I also differentiated between observability (the property) and observability tools (the technology). We'd need a new entry if we want to talk about the tech.   

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by CathPag — COMMENTED** [review]
Thanks, @idvoretskyi. Addressed your comments. Let me know what you think.

File: content/en/observability.md [hunk 1]
```diff
@@ -5,20 +5,12 @@ category: concept
...
-
-Observability is the capability to continuously generate and discover actionable insights based on signals from the system under observation. 
-In other words, observability allows users to understand a system's state from its external output and take (corrective) action.
-
-## Problem it addresses
+Observability is a system property that defines the degree to which the system can generate actionable insights. 
+It allows users to understand a system's state from these external outputs and take (corrective) action.
 
 Computer systems are measured by observing low-level signals such as CPU time, memory, disk space, and higher-level and business signals, including API response times, errors, transactions per second, etc.
+These observable systems are **observed** (or monitored) through specialized tools, so-called observability tools.
```
[the remark this answers is not in the document]
Comment (idvoretskyi) [reply content/en/observability.md:12]: Sounds good to me!

**Review by idvoretskyi — APPROVED** [review]
/lgtm

**Review by iamNoah1 — APPROVED** [review]

**Review by seokho-son — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**